client/deployment/src/main/resources/templates/libraries/microprofile/auth/compositeAuthenticationProvider.qute

```diff
@@ -1,9 +1,11 @@
 package {apiPackage}.auth;
 
 @jakarta.annotation.Priority(jakarta.ws.rs.Priorities.AUTHENTICATION)
-{#for auth in oauthMethods.orEmpty}
-@io.quarkiverse.openapi.generator.markers.OauthAuthenticationMarker(name="{auth.name}", openApiSpecId="{configKey}")
+{#if oauthMethods.orEmpty.size > 0}
```

Comment:
If this check really needed having into account that now we are removing duplicates?

Reply:
Unfortunately, we can't. `oauthMethods` not always is injected in the Qute template context by the underlying openapi-generator. In this cases, an exception would be thrown.